test/fuzz/lua/test_engine.lua

```diff
@@ -113,9 +132,17 @@ local err_pat_whitelist = {
     "Get%(%) doesn't support partial keys and non%-unique indexes",
     "Index '[%w_]+' %(RTREE%) of space '[%w_]+' %(memtx%) does not support max%(%)",
     "Index '[%w_]+' %(RTREE%) of space '[%w_]+' %(memtx%) does not support min%(%)",
-    "Failed to allocate %d+ bytes in [%w_]+ for [%w_]+",
+    "Failed to allocate %d+ bytes in [%w_ ]+ for [%w_]+",
     "Storage engine 'memtx' does not support cross%-engine transactions",
     "Storage engine 'vinyl' does not support cross%-engine transactions",
+    -- MEMCS
+    "Index '[%w_]+' %(TREE%) of space '[%w_]+' %(memcs%) does not support pagination",
+    "Arrow stream does not support field type '[%w_]+'",
+    "box_insert_arrow: field [%d]+ has unsupported type",
+    "Engine 'memcs' does not support upsert%(%)",
+    "Engine 'memcs' does not support index build",
+    "Engine 'memcs' does not support secondary indexes",
+    "Engine 'memcs' does not support variable field count",
```

Comment:
Isn't better to avoid using unsupported operations?

Reply:
Makes sense. I will add a new `engines` attributes to filter the operations.

Reply:
Now engine is checked in `insert_arrow_op`, `scan_column_op`, `arrow_stream_op`, but memcs-specific errors are still in `err_pat_whitelist`.